crates/edr_napi/src/context.rs

```diff
@@ -130,7 +130,7 @@ impl EdrContext {
         Ok(promise)
     }
 
-    #[doc = "Registers a new provider factory for the provided chain type."]
+    /// Registers a new provider factory for the provided chain type.
```

Comment:
Replaced doc attributes with doc comments as `cargo fmt` will automatically enforce line lengths for the latter.